internal/model/model.go

```diff
@@ -78,6 +78,8 @@ type Model struct {
 	detectedPackageName string
 	requestedFiles      map[string]bool
 	filteredLinesByFile map[string][]int
+    profilesLoaded      []*cover.Profile
+    sortByAsc           bool
```

Comment:
Instead of using two boolean flags, I would suggest using an "enum" type with values like `SortByPercentage`, `SortByStatements`, `SortByName` (default one), and a boolean flag that represents `asc` and `desc` values. Then, there will be a single key, not two keys as you suggested, that toggle over these 3 values:
- starting with order by name by default (if the flag is not set)
- toggle to sort by percentage
- optionally, if you with to implement as well, toggle to sort by uncovered statements
- toggle to sort by name again

So the point is that the user only uses a single key to toggle through all possible sorting states.
And then, there is another key that toggles between ascending and descending order. My thoughts were to use `s` key for sorting mode toggle, and `-` or `!` or `z` to reverse the order.

Of course if we drop the future requirement to implement 3 different sort modes (at least) your solution with a boolean flag is perfect, so if you don't feel like changing it, that's completely OK 😼

Reply:
I like your idea, and it will get me some practice using enums. Ill mess around with this and start working on some tests. Ill do frequent PRs so you can help guide me. Thanks for the feedback.